File: src/main/java/com/snowflake/snowpark_java/types/DayTimeIntervalType.java
package com.snowflake.snowpark_java.types;

/**
 * Day-time interval data type. This maps to INTERVAL DAY TO SECOND data type in Snowflake.
 *
 * <p>At the UDF/sproc boundary, values are transmitted as Arrow MonthDayNano and surfaced to Java
 * handlers as {@link java.time.Duration}.
 *
 * @since 1.14.0
 */
public class DayTimeIntervalType extends DataType {
  DayTimeIntervalType() {}
}

File: src/main/scala/com/snowflake/snowpark/types/DayTimeIntervalType.scala
package com.snowflake.snowpark.types

/**
 * Day-time interval data type. Mapped to INTERVAL DAY TO SECOND Snowflake data type.
 *
 * At the UDF/sproc boundary, day-time interval values are transmitted as Arrow MonthDayNano
 * ({months=0, days=D, nanoseconds=N}) and surfaced to Java/Scala handlers as
 * [[java.time.Duration]].
 *
 * @since 1.14.0
 */
object DayTimeIntervalType extends AtomicType
